tests/test_file.py

```diff
@@ -126,3 +131,142 @@ class Meta:
     assert not F().validate()
     assert not F(f=FileStorage()).validate()
     assert F(f=FileStorage(filename="real")).validate()
+    assert F(f=FileStorage(filename="real")).validate()
```

Comment:
This line is exactly the same as above :)

Reply:
Updated, thanks!